tests/test_encoder.py

```diff
@@ -66,7 +66,7 @@ def test_encoder(self):
     def test_with_wkt(self):
         self.assertRoundTrip(
             input_geometry="LINESTRING(-71.160281 42.258729,-71.160837 42.259113,-71.161144 42.25932)",  # noqa
-            expected_geometry=[[-71, 43], [-71, 43], [-71, 43]])
+            expected_geometry=[[-71, 42], [-71, 42], [-71, 42]])
```

Comment:
Wow. I didn't expect this... The new data makes sense. The old data... I can't understand.